### DISCUSSION TITLE
Flycast no muestra imagen des de la actualización del mes 02/23

### DISCUSSION THREAD

**Discussion by Melsanty** [flyinghead/flycast#971] [Q&A]
Banda una consulta por qué Flycast no me muestran imagen del Juego de Berserk y ni Shenmue 2 antes lo hacían pero ahora ya no

**Comment by flyinghead** [comment]
Rien n'a changé récemment dans ce domaine. Il est possible que ce soit dû à un problème réseau lors du téléchargement des images. Supprimer le répertoire data/boxart et relancer flycast.

**Reply by Melsanty** [reply]
Ya lo he echo como 4 veces y nada borre todo el APK los juegos y las bios y Nada
